.github/actions/constellation_create/gcp-logs.sh

```diff
@@ -3,49 +3,20 @@
 set -euo pipefail
 shopt -s inherit_errexit
 
-pushd constellation-terraform
-controlInstanceGroup=$(
-  terraform show -json |
-    jq -r .'values.root_module.child_modules[] |
-    select(.address == "module.instance_group_control_plane") |
-    .resources[0].values.base_instance_name'
-)
-workerInstanceGroup=$(
-  terraform show -json |
-    jq -r .'values.root_module.child_modules[] |
-    select(.address == "module.instance_group_worker") |
-     .resources[0].values.base_instance_name'
-)
-zone=$(
-  terraform show -json |
-    jq -r .'values.root_module.child_modules[] |
-    select(.address == "module.instance_group_control_plane") |
-     .resources[0].values.zone'
-)
-popd
+echo "Using Zone: ${1}"
+echo "Using Constellation UID: ${2}"
 
-controlInstances=$(
-  gcloud compute instance-groups managed list-instances "${controlInstanceGroup##*/}" \
-    --zone "${zone}" \
-    --format=json |
-    jq -r '.[] | .instance'
+allInstances=$(
+  gcloud compute instances list \
+    --filter="labels.constellation-uid=${2}" \
+    --format=json | yq '.[] | .id'
```

Comment:
```suggestion
    --format=json | yq '.[] | .name'
```
This should keep the name of the instance preserved, and also work when downloading the logs